dist/ci/travis_script.sh

```diff
@@ -46,15 +39,16 @@ if test -z "$SUBTEST"; then
       pushd ../backend
       make test_unit
       ;;
+    migrate)
+      mv db/structure.sql db/structure.sql.git
+      bundle exec rails db:drop db:create || exit 1
+      xzcat test/dump_2.5.sql.xz | sed '/^USE api_.*;/d' | mysql ci_api_test
```

Comment:
@adrianschroeter but in PR https://github.com/openSUSE/open-build-service/pull/3246 you said that you:

> synced the database setup in first migration with 2.5 database.

So why do we need the 2.5 database dump at all in this test? Just do `rake db:drop db:create db:migrate` and that will test the migrations from the 2.5 db?